From d5e91caaefa4191a9cb746f1933f8cdc5228c2da Mon Sep 17 00:00:00 2001
From: Bruce Ashfield <bruce.ashfield@windriver.com>
Date: Tue, 12 Sep 2017 19:57:11 -0400
Subject: [PATCH 1/6] aufs4: kbuild support

aufs4-kbuild.patch from git://github.com/sfjro/aufs4-standalone.git

Signed-off-by: Bruce Ashfield <bruce.ashfield@windriver.com>
---
 fs/Kconfig  | 1 +
 fs/Makefile | 2 ++
 2 files changed, 3 insertions(+)

diff --git a/fs/Kconfig b/fs/Kconfig
index 356cdf5f302c..fa8b53d96189 100644
--- a/fs/Kconfig
+++ b/fs/Kconfig
@@ -250,6 +250,7 @@ source "fs/pstore/Kconfig"
 source "fs/sysv/Kconfig"
 source "fs/ufs/Kconfig"
 source "fs/exofs/Kconfig"
+source "fs/aufs/Kconfig"
 
 endif # MISC_FILESYSTEMS
 
diff --git a/fs/Makefile b/fs/Makefile
index 73aa0ec31624..c39cd201fc50 100644
--- a/fs/Makefile
+++ b/fs/Makefile
@@ -129,3 +129,5 @@ obj-$(CONFIG_CEPH_FS)		+= ceph/
 obj-$(CONFIG_PSTORE)		+= pstore/
 obj-$(CONFIG_EFIVAR_FS)		+= efivarfs/
 obj-$(CONFIG_YAFFS_FS)		+= yaffs2/
+obj-$(CONFIG_AUFS_FS)           += aufs/
+
-- 
2.5.0

